src/providers/app-config/app-config.ts

```diff
@@ -34,6 +34,13 @@ export class AppConfigProvider {
       this.appConfig = {
```

Comment:
Perhaps we could do something about this config call failing? I imagine it's quite a common scenario (e.g. when offline).